anonymizer/OpenDiffix.Service/AggregationHooks.fs

```diff
@@ -189,20 +190,23 @@ module StarBucket =
           |> Value.unwrapBoolean
 
         if not isAlreadyMerged && isLowCount lowCountIndex bucket then
+          bucketCount <- bucketCount + 1
           bucket |> mergeAllAggregatorsInto starBucket
       )
 
     let executionContext = starBucket.ExecutionContext
 
-    // NOTE this is also true when there is no star bucket (no suppression)
-    //      this is because of how the initial value (`null`) of the
-    //      `DiffixLowCount` aggregator is translated to `Boolean true`
     let isStarBucketLowCount =
       starBucket.Aggregators.[lowCountIndex].Final(executionContext)
       |> Value.unwrapBoolean
 
+    let isStarBucketSingleBucket = bucketCount < 2
```

Comment:
Can also be zero buckets.

Reply:
yeah, I'll switch to `bucketCount = 1`, no point in making this inclusive

Reply:
If we do the single variable thing suggested above we avoid this fourth state.